package.json

```diff
@@ -13,6 +13,7 @@
     "@dnd-kit/core": "^6.0.8",
     "@dnd-kit/modifiers": "^6.0.1",
     "@dnd-kit/sortable": "^7.0.2",
+    "@emotion/is-prop-valid": "^1.1.1",
```

Comment:
nit: this helper is part of emotion package, I don't think we need to add it directly

```
> Found "@emotion/is-prop-valid@1.1.1"
info Reasons this module exists
   - "@emotion#styled" depends on it
   - Hoisted from "@emotion#styled#@emotion#is-prop-valid"
```